app/worker.js

```diff
@@ -120,6 +120,11 @@ class Worker {
       request.headers.set("Host", originalHost);
     }
 
+    request.headers.set("CF-CACHE-TTL", init.cf && init.cf.cacheTtl);
```

Comment:
should this be request.cf && request.cf.cacheTtl?

Reply:
Hi @gja, thanks for pointing this out. I believe the config is in `init` rather than part of the request. The client will call worker with, say `fetch(event.request, { cf: { cacheTtl: 300 } })`.
I also made some changes to make it more general as I think Cloudflare will likely support more cache rules with `init` in the future. 

Some CI still fails but seems unrelated to this PR. Maybe some npm dependencies are broken?